### ISSUE TITLE
Cannot get API to work

### ISSUE THREAD

**Issue by vic-t** [nilsteampassnet/TeamPass#2871]
### Steps to reproduce
Install and configure Teampass, create an API key, enable API, then open a browser and enter the following URL: 
https://**myTPsite**/api/index.php/info/version?apikey=**mykey**

### Expected behaviour
Return a json file

### Actual behaviour
404 Error

### Server configuration
**Operating system**:
Ubuntu 20.04 LTS
**Web server:**
Apache 2
**Database:**
MariaDB
**PHP version:**
7.4
**Teampass version:** 
Current beta 3
**Teampass configuration file:**
[tp.config.php.txt](https://github.com/nilsteampassnet/TeamPass/files/5900982/tp.config.php.txt)

**Updated from an older Teampass or fresh install:**
Fresh install

### Client configuration
**Browser:**
Firefox and Edge
**Operating system:**
Win10
#### Web server error log
No error in the logs about this.

**Comment by vic-t** [comment]
Hm, I guess I got a step further... just lose the "api" in the URL, and I actually get a result. So, that part of my report can be put down as documentation bug. Or is there a current documentation for version 3 that highlights the changes since version 2? 

Anyway, I got a new issue now. When I call the URL without the api in it, I'm being shown the following page: 
![grafik](https://user-images.githubusercontent.com/13408858/106401128-ba2b5600-6422-11eb-883b-b85e27b12310.png)
I assume something is not configured correctly. This looks wrong and none of the buttons seem to work anyway. Any idea why?
